desktop/core/src/desktop/conf.py

```diff
@@ -1818,6 +1818,16 @@ def get_instrumentation_default():
   default=False
 )
 
+def get_chunked_fileuploader():
+  return ENABLE_NEW_FILE_UPLOADER.get();
+
+ENABLE_NEW_FILE_UPLOADER = Config(
```

Comment:
Why use `NEW` here? Won't it ever become old? I would prefer calling it `ENABLE_CHUNKED_FILE_UPLOADER`. Am I missing something?

Reply:
changed it to `ENABLE_CHUNKED_FILE_UPLOADER`